### PR TITLE
Added RT lead shadows for 1.18

### PR DESCRIPTION
**Pull request by alejandrox1**
xref: https://github.com/kubernetes/sig-release/issues/933

Signed-off-by: alejandrox1 <alarcj137@gmail.com>

### CODE REVIEW

File: releases/release-1.18/release_team.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 | **Role** | **Name** (**GitHub / Slack ID**)  | **Shadow Name(s) (GitHub / Slack ID)** |
 | ------ | ------ | ------ |
-| Lead |  Jorge Alarcon ([@alejandrox1](https://github.com/alejandrox1) / Slack: `@alejandrox1`) |  |
+| Lead |  Jorge Alarcon ([@alejandrox1](https://github.com/alejandrox1) / Slack: `@alejandrox1`) | Damini Satya ([@daminisatya](https://github.com/daminisatya) / Slack: `@Damini Satya`), Bob Killen ([@mrbobbytables](https://github.com/mrbobbytables) / Slack: `@@mrbobbytables`)  |
```
Comment (justaugustus) [inline releases/release-1.18/release_team.md]:
```suggestion
| Lead |  Jorge Alarcon ([@alejandrox1](https://github.com/alejandrox1) / Slack: `@alejandrox1`) | Damini Satya ([@daminisatya](https://github.com/daminisatya) / Slack: `@Damini Satya`), Bob Killen ([@mrbobbytables](https://github.com/mrbobbytables) / Slack: `@mrbobbytables`)  |
```

**Comment by alejandrox1** [comment]
Thank you for the review @justaugustus 
updated this PR

File: releases/release-1.18/release_team.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 | **Role** | **Name** (**GitHub / Slack ID**)  | **Shadow Name(s) (GitHub / Slack ID)** |
 | ------ | ------ | ------ |
-| Lead |  Jorge Alarcon ([@alejandrox1](https://github.com/alejandrox1) / Slack: `@alejandrox1`) |  |
+| Lead | Jorge Alarcon ([@alejandrox1](https://github.com/alejandrox1) / Slack: `@alejandrox1`) | Damini Satya ([@daminisatya](https://github.com/daminisatya) / Slack: `@Damini Satya`), Bob Killen ([@mrbobbytables](https://github.com/mrbobbytables) / Slack: `@@mrbobbytables`)  |
```
Comment (justaugustus) [inline releases/release-1.18/release_team.md]:
```suggestion
| Lead | Jorge Alarcon ([@alejandrox1](https://github.com/alejandrox1) / Slack: `@alejandrox1`) | Damini Satya ([@daminisatya](https://github.com/daminisatya) / Slack: `@Damini Satya`), Bob Killen ([@mrbobbytables](https://github.com/mrbobbytables) / Slack: `@mrbobbytables`)  |
```

Still has the double `@`.

Comment (alejandrox1) [reply releases/release-1.18/release_team.md]: ha didn't see that bit.
ptal 
/cc @justaugustus

**Pull request merged**